_rules/auto-play-audio-has-control-mechanism-4c31df.md

```diff
@@ -0,0 +1,226 @@
+</head>
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay="true">
+		 <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	   	 <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls">
+			<button type="button" id="play-pause" class="play">Play</button>
+			<button type="button" id="mute">Mute</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 4
+
+The `<video>` has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off, but the control `<button>` elements do not have accessible names.
+
+``` html
+<head>
+<style>
+ button {color: #000;}
+ button:hover {cursor: pointer;	cursor: pointer; background-color: grey;  color: white;}
+</style>
+</head>
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay="true">
+		  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	   	  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls">
+      		<button type="button" id="play-pause" class="play"></button>
+		<button type="button" id="mute"></button>
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 5
+
+The `<video>` has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off, but the mechanism is not included in the accessibility tree.
+
+``` html
+<head>
+<style>
+button {color: #000;}
+button:hover {cursor: pointer;	cursor: pointer; background-color: grey;  color: white;}
+</style>
+</head>
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay="true">
+		 <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	   	 <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls" aria-hidden="true">
+			<button type="button" id="play-pause" class="play">Play</button>
+			<button type="button" id="mute">Mute</button>
+		</div>
+	</div>
+</body>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The `<video>` element is `muted`.
+
+``` html
+ <video autoplay="true" muted="true">
+  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+#### Inapplicable Example 2
+
+The `<video>` element `src` file has no audio output.
+
+``` html
+ <video autoplay="true">
+  <source src="../test-assets/rabbit-video/video-with-incorrect-voiceover.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video-with-incorrect-voiceover.webm" type="video/webm" />
+</video>
+```
+
+#### Inapplicable Example 3
+
+The `audio` element does not autoplay.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" controls></audio>
```

Comment:
Can you run prettier on this PR please.

Reply:
@jkodu give me a shout when this is done. Thanks for helping me out!!